server/views/pages/createGoals/index.njk

```diff
@@ -171,15 +172,17 @@
                   </strong>
                 </div>
                 {{ errors | findError('note') }}
-                <textarea class="govuk-textarea" id="note" name="note" rows="5" aria-describedby="warning-text">{{ form.note }}</textarea>
+                <textarea class="govuk-textarea" id="note" name="note" rows="5" aria-describedby="warning-text">{{ form.goals[0].note }}</textarea>
               </div>
             </div>
 
-
           </div>
         </div>
 
         {{ govukButton({
+          id: "submit-button",
+          name: "submit-button",
```

Comment:
I think this needs to be `name: "action"` so that in the controller code (yet to be written) we will know the difference between when the user presses the main Submit CTA (this button) vs. add another step vs. add another goal (not being added in this story, but will come in a subsequent story)